common/persistence/nosql/nosqlplugin/cassandra/gocql/client.go

```diff
@@ -158,6 +159,18 @@ func NewCassandraCluster(
 	}
 
 	cluster.PoolConfig.HostSelectionPolicy = gocql.TokenAwareHostPolicy(gocql.RoundRobinHostPolicy())
+
+	if cfg.AddressTranslator != nil && cfg.AddressTranslator.Translator != "" {
+		addressTranslator, err := translator.LookupTranslator(cfg.AddressTranslator.Translator)
+		if err != nil {
+			return nil, err
+		}
+		cluster.AddressTranslator, err = addressTranslator.GetTranslator(&cfg)
```

Comment:
@yiminc I need to set this `AddressTranslator` field on `cluster`, spefically. This is gocql stuff.